edx/frontend-app-discussions
# feat: Add topic relation to course structure [BD-38] [TNL-8790]

Refactors the data code related to topics and uses the course blocks API to get the LMS link for a topic so it can be embedded in the post.

**JIRA tickets**:
- https://openedx.atlassian.net/browse/TNL-8790

**Dependencies**: https://github.com/edx/frontend-app-discussions/pull/28

**Screenshots**: 
![Screenshot 2021-10-28 at 08-15-42 Discussions localhost](https://user-images.githubusercontent.com/118837/139215534-2dbd5f6b-bf6b-4986-9400-64e10905c003.png)

**Merge deadline**: "None"

**Testing instructions**:

1. Create a new post in an in-context discussion
2. Visit this post and see that the post displays the category and topic name and a link to the LMS. 

**Reviewers**
